src/dom/ui-manip.js

```diff
@@ -0,0 +1,106 @@
+import { format, parse } from "date-fns";
+
+const spinnerContainer = document.getElementById("all");
+const detailsContainer = document.getElementById("heading");
+const buttonContainer = document.getElementById("select-button");
+const racesContainer = document.getElementById("select-race");
+const lapDetailsContainer = document.getElementById("lap-details");
+
+export default function addLapButton(btnNum, runsJSON) {
```

Comment:
Consider adding the callback for on click as a param.
e.g. 
```js
export default function addLapButton(btnNum, runsJSON, onClickHandler) {
  const button = document.createElement("button");
  // ...
  button.addEventListener('click', onClickHandler);
  buttonContainer.appendChild(button);
  
  return button;
}
```

Reply:
Perhaps not as there is more than one event the creator of this element may want to bind callbacks to.
e.g. On Enter pressed while focussed.